core/domain/collection_services.py

```diff
@@ -236,13 +294,26 @@ def get_multiple_collections_by_id(collection_ids, strict=True):
 
 
 def get_new_collection_id():
-    """Returns a new collection id."""
+    """Returns a new collection id.
+
+    Returns:
+        A new colleciton id.
+    """
     return collection_models.CollectionModel.get_new_id('')
 
 
 def is_collection_summary_editable(collection_summary, user_id=None):
     """Checks if a given user may edit an collection by checking
     the given domain object.
+
+    Args:
+        collection_summary: A collection summary domain object that the user
+        with the given id is editing
```

Comment:
Indent by 4. Drop newline below this one (try to keep everything looking consistent).